### ISSUE TITLE
New page loads are not announced to users using assistive technologies

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#97]
When a user activates a link that causes client-side navigation, assistive technologies are not notified of the navigation event. Therefore:

1. A screen reader will still read out content from the previous view until the user tabs/navigates to a different area on the page
2. When new content is loaded, the user is not informed that the new content is loaded
3. The new focus state is not consistent, often leaving the user lost

To resolve this issue we need to actively manage focus on navigation(such as discussed in Bug 1563471 - https://bugzilla.mozilla.org/show_bug.cgi?id=1563471) and possibly make use of Aria live-regions to inform the user when content has changed.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563511

**Comment by schalkneethling** [comment]
Pull request: https://github.com/mozilla/kuma/pull/5531
